lib/shoulda/matchers/action_controller/store.rb

```diff
@@ -0,0 +1,36 @@
+module Shoulda
+  module Matchers
+    module ActionController
+      # @private
+      class Store
+        NO_KEY_SET = Object.new
+        private_constant :NO_KEY_SET
+
+        attr_accessor :controller
+
+        def name
+          raise NotImplementedError, "must be implemented by subclass"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.